### PR TITLE
Fixes various audio enums, fixes some issues in audio code.

### PR DESCRIPTION
**Pull request by tomsons26**

### CODE REVIEW

File: src/game/common/audio/audiomanager.cpp [hunk 1]
```diff
@@ -538,6 +538,7 @@ int AudioManager::Add_Audio_Event(const AudioEventRTS *event)
         Get_Info_For_Audio_Event(event);
 
         if (event->Get_Event_Info() == nullptr) {
+            captainslog_debug("No info for requested audio event '%s'\n", event->Get_Event_Name());
```
Comment (xezon) [inline src/game/common/audio/audiomanager.cpp]: \n can be omitted.

File: src/game/common/audio/audiomanager.cpp [hunk 2]
```diff
@@ -586,44 +587,44 @@ int AudioManager::Add_Audio_Event(const AudioEventRTS *event)
...
+    } else {
+        m_soundManager->Add_Audio_Event(new_event);
+    }
+
+    if (new_event != nullptr) {
+        return new_event->Get_Playing_Handle();
+    }
 
-    return 3; // Don't play locally?
+    // Is this even possible to reach?
```
Comment (xezon) [inline src/game/common/audio/audiomanager.cpp]: Code paths now look well defined and this comment can be removed.

**Review by xezon — APPROVED** [review]
Logically looks good.

File: src/game/common/audio/audiorequest.h [hunk 3]
```diff
@@ -26,9 +26,9 @@ union reqevent_t
 
 enum RequestType
```
Comment (xezon) [inline src/game/common/audio/audiorequest.h]: `enum AudioRequestType`

File: src/game/common/audio/audioeventinfo.h [hunk 4]
```diff
@@ -23,11 +23,19 @@
 
 enum AudioType : int32_t
```
Comment (xezon) [inline src/game/common/audio/audioeventinfo.h:24]: `enum AudioEventType`

**Review by xezon — APPROVED** [review]
Looks good.

File: src/game/common/audio/audioeventinfo.h (hunk 4 above)
Comment (tomsons26) [reply src/game/common/audio/audioeventinfo.h:24]: not quite, there's a unclarity on if its EVENT actually, for now leaving as is

**Pull request merged**